store/tikv/batch_coprocessor.go

```diff
@@ -130,6 +130,8 @@ func buildBatchCopTasks(bo *Backoffer, cache *RegionCache, ranges *copRanges, re
 			// of date and already be cleaned up. We should retry and generate new tasks.
 			if rpcCtx == nil {
 				needRetry = true
+				bo.Backoff(BoRegionMiss, errors.New("Cannot find region or TiFlash peer"))
```

Comment:
maybe need check Backoff return value and return error if err != nil

Reply:
yes, I find it :P